models/models.go

```diff
@@ -114,6 +115,15 @@ var (
 	ErrorNoData      = errors.New("Bad request - Missing data in body")
 )
 
+// ErrorDuplicateDimension is returned if a request contains a duplicate dimension
+type ErrorDuplicateDimension struct {
```

Comment:
Naming convention of error types should end with `Error` - see [here](https://github.com/golang/go/wiki/Errors#naming)